clients/privacy-center/cypress/e2e/privacy-request.cy.ts

```diff
@@ -13,58 +13,24 @@ describe("Privacy request", () => {
       ).as("postPrivacyRequestVerify");
     });
 
-    it("can verify phone and navigate to form", () => {
```

Comment:
Blah, this is unfortunate... can't we wire up a different config to test this? I'm pretty sure our test harnesses support swapping in different config.json?

Reply:
I was able to get this working via an env variable to switch on which config.json is loaded in. I didn't find any way to do this within cypress itself.

Reply:
Hm, nvm, seems like using the env var is causing some unintended issues in github ci (not locally)- https://github.com/ethyca/fides/actions/runs/4632049188/jobs/8195643338?pr=2539#step:5:36.